lib/room.rb

```diff
@@ -1,12 +1,24 @@
-class Rom
-
+class Room
+  attr_reader :category,
+              :length,
+              :width,
+              :painted
```

Comment:
Since we don't ask for methods `length`, `width` or `paint`, these do not need to be attr_readers.